### PR TITLE
Sort the libraries

### PR DESCRIPTION
**Pull request by ctide**
This just alphabetically sorts the list of libraries, so I'm not digging through the list of one off libraries for underscore every time.

### CODE REVIEW

File: public/js/editors/libraries.js [hunk 1]
```diff
@@ -688,7 +688,7 @@ var libraries = [
   }
 ];
 
-window.libraries = libraries; // expose a command line API
+window.libraries = _.sortBy(libraries, function(lib) { return lib.label; }); // expose a command line API
```
Comment (remy) [inline public/js/editors/libraries.js]: Is underscore floating around in the globals somewhere?! (...I'm not even sure it's intentionally pulled in!)

**Comment by ctide** [comment]
Woops, you are correct. Switched it to just use plain old array#sort.

**Comment by remy** [comment]
lol, totally confused me there for a sec!

So, as much as this is good (I :heart: PRs), won't this bust the ordering of the groups? I've not tested it, but I suspect it will, mixing them altogether...

**Comment by ctide** [comment]
Is that ordering important? It seemed pretty haphazard to me (lots of
'dead' scripts higher up in the list, for example), which is why I
disregarded it.

**Comment by remy** [comment]
I think you're referring to the "end" grouping (i.e. ungrouped). That all said, it might be worth parking this in lieu of #2531 landing. That okay?

**Comment by ctide** [comment]
Oh, yeah, that works too. I was trying to do the least intrusive change
here, since I ran into a slew of node package issues trying to get it
running locally so I was mostly just winging it. :)

**Pull request closed without merging**